HopsanGenerator/src/GeneratorTypes.cpp

```diff
@@ -240,6 +240,9 @@ bool ComponentLibrary::loadFromXML(QString filepath)
         else if (elementName == "source") {
             mSourceFiles.append(reader.readElementText());
         }
+        else if (elementName == "extrasource") {
```

Comment:
Here extrasource is loaded from XML, but it does not seem to be saved in the save to XML function
Also the HoLC code may need to be update to set this field properly (or just empty)

Reply:
Done and done. I don't want to add too much new functionality to HoLC since we are going to merge it into HopsanGUI, but now it can at least handle extra source files.

Reply:
Good, then that field wont be lost when you open a library in holc as it will save the XML file overwriting the old one